### ISSUE TITLE
Volume Control

### ISSUE THREAD

**Issue by jaypatel512** [skiselkov/BetterPushbackC#148]
Hey @skiselkov ! First of all, thank you so much for your awesome plugin.

I feel greedy asking more feature from this plugin, but one thing that bothers me most, is the volume level. Not that I love to hear your voice (I have repeatedly watched your youtube videos), but sometimes the audio level is too high, and it sometimes interfere with ATC communications.

I would love to create a PR or something to get you started, but I may need few pointers to setup and compile your project.

**Comment by jaypatel512** [comment]
Btw, I just realized, I can use the `radio level` from SIM to control the audio level, but wouldn't that also reduce my ATC volume ?

https://github.com/skiselkov/BetterPushbackC/blob/master/src/msg.c#L261

**Comment by skiselkov** [comment]
Yes, it would reduce your ATC volume (well, probably, depending on what ATC plugin you're using). I'll have a look at creating a separate slider to control tug radio volume in case the default binding to X-Plane's radio volume doesn't work.
